### ISSUE TITLE
Passwordless login fails to work after upgrade from 2023.10.7 to 2024.2.0

### ISSUE THREAD

**Issue by bigdaddy5414** [goauthentik/authentik#8631]
**Describe the bug**
Passwordless login fails to work after upgrade from 2023.10.7 to 2024.2.0 It just states there was a general error.

**To Reproduce**
1. Go to 'Use a security key'
2. Something went wrong! Please try again later.

**Expected behavior**
Uses FACEID and logs in automatically

**Logs**
<details>
    <summary>Stacktrace from authentik</summary>

```
Traceback (most recent call last):
  File "/authentik/flows/views/executor.py", line 291, in get
    stage_response = self.current_stage_view.dispatch(request)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.12/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 243, in get
    challenges = self.get_webauthn_challenge_without_user()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/stage.py", line 216, in get_webauthn_challenge_without_user
    "challenge": get_webauthn_challenge_without_user(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/authentik/stages/authenticator_validate/challenge.py", line 72, in get_webauthn_challenge_without_user
    return loads(options_to_json(authentication_options))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/ak-root/venv/lib/python3.12/site-packages/webauthn/helpers/options_to_json.py", line 118, in options_to_json
    auth_to_return["userVerification"] = options.user_verification.value
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
builtins.AttributeError: 'str' object has no attribute 'value'
```
</details>

**Version and Deployment (please complete the following information):**
- authentik version: 2024.2.0
- Deployment: [e.g. docker-compose, helm]

**Comment by FML128** [comment]
Same issue here

**Comment by freender** [comment]
Same issue:

`"Traceback (most recent call last):\n  File \"/authentik/flows/views/executor.py\", line 291, in get\n    stage_response = self.current_stage_view.dispatch(request)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/ak-root/venv/lib/python3.12/site-packages/django/views/generic/base.py\", line 143, in dispatch\n    return handler(request, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/stage.py\", line 243, in get\n    challenges = self.get_webauthn_challenge_without_user()\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/stage.py\", line 216, in get_webauthn_challenge_without_user\n    \"challenge\": get_webauthn_challenge_without_user(\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/challenge.py\", line 72, in get_webauthn_challenge_without_user\n    return loads(options_to_json(authentication_options))\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/ak-root/venv/lib/python3.12/site-packages/webauthn/helpers/options_to_json.py\", line 118, in options_to_json\n    auth_to_return[\"userVerification\"] = options.user_verification.value\n                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nbuiltins.AttributeError: 'str' object has no attribute 'value'"} `

**Comment by SirBomble** [comment]
Same issue here. 

`server-1      | {"auth_via": "unauthenticated", "domain_url": "REDACTED", "event": "AttributeError(\"'str' object has no attribute 'value'\")", "flow_slug": "passwordless-flow", "host": "REDACTED", "level": "warning", "logger": "authentik.flows.views.executor", "pid": 68, "request_id": "3e77b9b889a04dda94c08668e7cc8101", "schema_name": "public", "timestamp": "2024-02-22T02:18:18.162299"}
server-1      | {"action": "system_exception", "auth_via": "unauthenticated", "client_ip": "REDACTED", "context": {"asn": {"as_org": "REDACTED", "asn": "REDACTED", "network": "REDACTED"}, "geo": {"city": "REDACTED", "continent": "NA", "country": "US", "lat": "REDACTED", "long": "REDACTED"}, "http_request": {"args": {"next": "/"}, "method": "GET", "path": "/api/v3/flows/executor/passwordless-flow/", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36"}, "message": "Traceback (most recent call last):\n  File \"/authentik/flows/views/executor.py\", line 291, in get\n    stage_response = self.current_stage_view.dispatch(request)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/ak-root/venv/lib/python3.12/site-packages/django/views/generic/base.py\", line 143, in dispatch\n    return handler(request, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/stage.py\", line 243, in get\n    challenges = self.get_webauthn_challenge_without_user()\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/stage.py\", line 216, in get_webauthn_challenge_without_user\n    \"challenge\": get_webauthn_challenge_without_user(\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/challenge.py\", line 72, in get_webauthn_challenge_without_user\n    return loads(options_to_json(authentication_options))\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/ak-root/venv/lib/python3.12/site-packages/webauthn/helpers/options_to_json.py\", line 118, in options_to_json\n    auth_to_return[\"userVerification\"] = options.user_verification.value\n                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nbuiltins.AttributeError: 'str' object has no attribute 'value'"}, "domain_url": "REDACTED", "event": "Created Event", "host": "REDACTED", "level": "info", "logger": "authentik.events.models", "pid": 68, "request_id": "3e77b9b889a04dda94c08668e7cc8101", "schema_name": "public", "timestamp": "2024-02-22T02:18:18.182119", "user": {"email": "", "is_anonymous": true, "pk": 2, "username": "AnonymousUser"}}`

**Comment by DavidFuchs** [comment]
Exact same issue here. I just installed Authentik last night, mind you, so not a lot of experience here.

If I enter my username and password first, it then prompts me for my webauthn credential and everything works fine. But if I go straight to the passwordless flow, I get the `Something went wrong! Please try again later.` error.

I followed the instructions here: https://www.youtube.com/watch?v=aEpT2fYGwLw

```
2024-02-22T03:29:57.442712611Z INF | auth_via=unauthenticated domain_url=[redacted] event=/if/flow/df-passwordless-flow/?next=%2F host=[redacted] logger=authentik.asgi method=GET pid=19182 remote=10.0.0.1 request_id=0bc243ce92544150bbd4874262784c36 runtime=7 schema_name=public scheme=https status=200 timestamp=2024-02-22T03:29:57.442573 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 Edg/121.0.0.0 
2024-02-22T03:29:57.499477886Z INF | domain_url=null event=/ws/client/ logger=authentik.asgi pid=19182 remote=10.0.0.1 schema_name=public scheme=ws timestamp=2024-02-22T03:29:57.499351 user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 Edg/121.0.0.0 
2024-02-22T03:29:57.513246454Z warning | auth_via=unauthenticated domain_url=[redacted] event=AttributeError("'str' object has no attribute 'value'") flow_slug=df-passwordless-flow host=[redacted] logger=authentik.flows.views.executor pid=19182 request_id=6ddf304b28ae4ee39922f0697832f63d schema_name=public timestamp=2024-02-22T03:29:57.513117 
2024-02-22T03:29:57.515484049Z INF | action=system_exception auth_via=unauthenticated client_ip=10.0.0.1 context={"http_request":{"args":{"next":"/"},"method":"GET","path":"/api/v3/flows/executor/df-passwordless-flow/","user_agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 Edg/121.0.0.0"},"message":"Traceback (most recent call last):\n  File \"/authentik/flows/views/executor.py\", line 291, in get\n    stage_response = self.current_stage_view.dispatch(request)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/ak-root/venv/lib/python3.12/site-packages/django/views/generic/base.py\", line 143, in dispatch\n    return handler(request, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/stage.py\", line 243, in get\n    challenges = self.get_webauthn_challenge_without_user()\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/stage.py\", line 216, in get_webauthn_challenge_without_user\n    \"challenge\": get_webauthn_challenge_without_user(\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/authentik/stages/authenticator_validate/challenge.py\", line 72, in get_webauthn_challenge_without_user\n    return loads(options_to_json(authentication_options))\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/ak-root/venv/lib/python3.12/site-packages/webauthn/helpers/options_to_json.py\", line 118, in options_to_json\n    auth_to_return[\"userVerification\"] = options.user_verification.value\n                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nbuiltins.AttributeError: 'str' object has no attribute 'value'"} domain_url=[redacted] event=Created Event host=[redacted] logger=authentik.events.models pid=19182 request_id=6ddf304b28ae4ee39922f0697832f63d schema_name=public timestamp=2024-02-22T03:29:57.515383 user={"email":"","is_anonymous":true,"pk":1,"username":"AnonymousUser"} 
```

**Comment by BeryJu** [comment]
duplicate of #8618, will be fixed by 2024.2.1
